core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -1,25 +1,4 @@
 <script type="text/ng-template" id="skins/Conversation">
-  <!--
```

Comment:
It is moved to tutor_card_directive.html along with the code that handles the animation.